src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -198,6 +198,16 @@ private function addSelect(QueryBuilder $queryBuilder, string $entity, string $a
                 if (true === $targetClassMetadata->hasField($property) && (true === $propertyMetadata->getAttribute('fetchable') || true === $propertyMetadata->isReadable())) {
```

Comment:
IMO strict checking is always good. I'd keep the strict equality on the `getAttribute` part. For the rest it's up to you as they should return booleans.